app/routes/events/components/EventAdministrate/AdminRegisterForm.js

```diff
@@ -23,18 +23,21 @@ const AdminRegister = ({
       <form onSubmit={handleSubmit}>
         <Field
           placeholder="Begrunnelse"
+          label="Begrunnelse"
           name="reason"
           component={TextEditor.Field}
         />
         <Field
           placeholder="Tilbakemelding"
+          label="Tilbakemelding"
           name="feedback"
           component={TextEditor.Field}
         />
         <Field
           name="pool"
           component={SelectInput.Field}
           placeholder="Pool"
+          label="Pool"
```

Comment:
I think we should call pool something more understandable to the user - "Påmeldingsgruppe" or something like that? cc @edvardvb @hanswilw

Reply:
Hmm, I prefer 'Pulje', but it might not be very understandable either

Reply:
I'll merge, then we can fix later.